src/common/config/decrypt.tsx

```diff
@@ -1,11 +1,12 @@
-import { Wallet, providers } from "ethers";
-import { AwsKmwSignerOption, ConfigFile, ConnectedSigner } from "../../types";
+import { ProviderDetails, utils } from "@govtechsg/oa-verify";
 import { RelayProvider } from "@opengsn/gsn";
```

Comment:
Woah.. We have GSN here?🤣

Reply:
haha, i think was previously added in but not used